vue2-vuetify/src/complex/components/AdditionalProperties.vue

```diff
@@ -0,0 +1,463 @@
+<template>
+  <v-card v-if="control.visible" elevation="0">
+    <v-card-title>
+      <v-toolbar flat>
+        <v-toolbar-title>{{ additionalPropertiesTitle }}</v-toolbar-title>
+        <v-spacer></v-spacer>
+
+        <v-hover v-slot="{ hover }">
+          <v-text-field
+            v-disabled-icon-focus
+            :required="true"
+            :class="styles.control.input"
+            :error-messages="errors"
+            v-model="newPropertyName"
+            :clearable="hover"
+            :placeholder="placeholder"
+            :disabled="!control.enabled"
+            v-bind="vuetifyProps('v-text-field')"
+          >
+          </v-text-field>
+        </v-hover>
+        <v-tooltip bottom>
+          <template v-slot:activator="{ on: onTooltip }">
+            <v-btn
+              fab
+              text
+              elevation="0"
+              small
+              :aria-label="addToLabel"
+              v-on="onTooltip"
+              :disabled="
+                !control.enabled ||
+                (appliedOptions.restrict &&
+                  control.schema.maxProperties !== undefined &&
+                  control.data &&
+                  Object.keys(control.data).length >=
+                    control.schema.maxProperties) ||
+                errors.length > 0 ||
+                !newPropertyName
```

Comment:
This computation is rather complicated and hard to read. Could you move it to code instead of instead of embedding it in the template and maybe add a short comment?

Reply:
@kchobantonov Please have a look again at this comment. I think it was hidden earlier.

Reply:
added in the last commit